recipe/meta.yaml

```
@@ -60,6 +60,8 @@ outputs:
         - cython                                 # [build_platform != target_platform]
         - numpy                                  # [build_platform != target_platform]
         - gnuconfig                              # [osx and arm64]
+        # Hack to try to align grpc packages across build and host for Windows?
+        - openssl 1
```

Should this have a Windows selector?

```suggestion
        # Hack to try to align grpc packages across build and host for Windows?
        - openssl 1  # [win]
```

---

Doesn't look like this is fixing it anyway so I'll remove it. Very much just hacking on this, but if you have any ideas on resolving the grpc linking errors I'd welcome them.